### PR TITLE
Added HUD

### PR DESCRIPTION
**Pull request by Sebjen0209**

### CODE REVIEW

**Comment by Nolle10** [comment]
Just a quick reminder for tomorrow, we should add a DecimalFormatter to the doubles, since they can get quite large.

File: src/main/java/worldofzuul/presentation/SceneController.java [hunk 1]
```diff
@@ -31,8 +27,23 @@
...
+    private Label moneyLabel, emissionLabel, energyProductionLabel, yearLabel;
+
     @FXML
     private void initialize() {
+        if(yearLabel != null){
+            yearLabel.getText();
+            emissionLabel.getText();
+            energyProductionLabel.getText();
+            moneyLabel.getText();
+            yearLabel.setText(String.format("Year: " + (2010 + Game.instance.getGameYear())));
```
Comment (mathiasgredal) [inline src/main/java/worldofzuul/presentation/SceneController.java]: Why are you using string concatenation inside string.format?

**Review by mathiasgredal — CHANGES_REQUESTED** [review]
Almost good, you just need to fix some of the string formatting issues

**Review by mathiasgredal — APPROVED** [review]
LGTM(Lets get this merged)

**Pull request merged**